label_sync/labels.md

```diff
@@ -98,7 +99,7 @@ larger set of contributors to apply/remove them.
 | <a id="sig/multicluster" href="#sig/multicluster">`sig/multicluster`</a> | Categorizes an issue or PR as relevant to sig-multicluster. <br><br> This was previously `sig/federation`, `sig/federation (deprecated - do not use)`, | anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
 | <a id="sig/network" href="#sig/network">`sig/network`</a> | Categorizes an issue or PR as relevant to sig-network.| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
 | <a id="sig/node" href="#sig/node">`sig/node`</a> | Categorizes an issue or PR as relevant to sig-node.| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
-| <a id="sig/onprem" href="#sig/onprem">`sig/onprem`</a> | REMOVING. This will be deleted after 2019-12-17 00:00:00 +0000 UTC <br><br> Categorizes an issue or PR as relevant to sig-onprem.| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
+| <a id="sig/onprem" href="#sig/onprem">`sig/onprem`</a> | REMOVING. This will be deleted after 2019-12-17 00:00:00 +0000 +0000 <br><br> Categorizes an issue or PR as relevant to sig-onprem.| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
```

Comment:
There's a bug in the generator script causing this date to get mangled.
We can probably just drop this label given the note:
> REMOVING. This will be deleted after 2019-12-17 00:00:00 +0000 +0000 <br><br> Categorizes an issue or PR as relevant to sig-onprem.

Reply:
there are other instances of mangled deleteAfter dates... I'll leave that cleanup in the generator for a follow up

Reply:
>  I'll leave that cleanup in the generator for a follow up

or... maybe I won't... hooray for inconsistent generated results:

```
Validating labels...
Updating labels...
< unexpected
> missing
102c102
< | <a id="sig/onprem" href="#sig/onprem">`sig/onprem`</a> | REMOVING. This will be deleted after 2019-12-17 00:00:00 +0000 +0000 <br><br> Categorizes an issue or PR as relevant to sig-onprem.| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
---
> | <a id="sig/onprem" href="#sig/onprem">`sig/onprem`</a> | REMOVING. This will be deleted after 2019-12-17 00:00:00 +0000 UTC <br><br> Categorizes an issue or PR as relevant to sig-onprem.| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |

ERROR: labels.md out of date. Fix with hack/update-labels.sh
```

Reply:
removed